doc/man/8/radosgw-admin.rst

```diff
@@ -424,9 +768,9 @@ Options
 
 	List of caps (e.g., "usage=read, write; user=read".
 
-.. option:: --compression=<compression-algorithm>
+.. option:: --compression=<type>
 
-    Placement target compression algorithm (lz4|snappy|zlib|zstd)
+    Placement target compression type (plugin name or empty/none).
```

Comment:
here too

Reply:
I was making it similar to the text in rgw_admin.cc. Anyway, reverted it.